interface/forms/eye_mag/view.php

```diff
@@ -162,134 +167,25 @@
-    }
-     // This invokes the find-code popup.
-    function sel_diagnosis(target,term) {
-      if (target =='') {
-          target = "0";
-      }
-      IMP_target = target;
-        <?php
-        if ($irow['type'] == 'PMH') { //or POH
-        ?>
-          dlgopen('<?php echo $rootdir ?>/patient_file/encounter/find_code_popup.php?codetype=<?php echo attr(collect_codetypes("medical_problem", "csv")) ?>&search_term='+encodeURI(term), '_blank', 600, 400);
-            <?php
-        } else {
-            ?>
-          dlgopen('<?php echo $rootdir ?>/patient_file/encounter/find_code_popup.php?codetype=<?php echo attr(collect_codetypes("diagnosis", "csv")) ?>&search_term='+encodeURI(term), '_blank', 600, 400);
-            <?php
-        }
-        ?>
-    }
-
-    var obj =[];
-    <?php
-      //also add in any obj.Clinical data if the form was already opened
-      $codes_found = start_your_engines($encounter_data);
-    if ($codes_found) { ?>
-      obj.Clinical = [<?php echo json_encode($codes_found[0]); ?>];
-    <?php
-    } ?>
-
-    </script>
+      <title> <?php echo xlt('Chart'); ?>: <?php echo text($pat_data['fname'])." ".text($pat_data['lname'])." ".text($visit_date); ?></title>
+      <link rel="shortcut icon" href="<?php echo $GLOBALS['images_static_relative']; ?>/favicon.ico" />
+      <meta charset="utf-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="description" content="OpenEMR: Eye Exam">
+      <meta name="author" content="OpenEMR: Ophthalmology">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <!-- Add Font stuff for the look and feel.  -->
+      <?php Header::setupHeader([ 'jquery-ui', 'jquery-ui-redmond','datetime-picker', 'dialog' ,'jscolor', 'qtip2' ]); ?>
+      <!-- Add Font stuff for the look and feel.  -->
 
-    <link rel="stylesheet" href="<?php echo $GLOBALS['css_header']; ?>" type="text/css">
-    <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/jquery-ui-1-11-4/themes/excite-bike/jquery-ui.css">
-    <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/bootstrap-3-3-4/dist/css/bootstrap.min.css">
-    <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/pure-0-5-0/pure-min.css">
-    <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/qtip2-2-2-1/jquery.qtip.min.css" />
-    <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/font-awesome-4-6-3/css/font-awesome.min.css">
-    <link rel="stylesheet" href="../../forms/<?php echo $form_folder; ?>/css/style.css?v=<?php echo $v_js_includes; ?>" type="text/css">
+      <link rel="stylesheet" href="../../forms/<?php echo $form_folder; ?>/css/style.css?v=<?php echo $v_js_includes; ?>" type="text/css">
+      <Xscript src="<?php echo $GLOBALS['assets_static_relative'] ?>/jquery-min-1-10-2/index.js"></Xscript>
```

Comment:
Is this X a typo?

Reply:
No I am deleting it and using the Header jquery.  Next review it is gone.